content/docs/instrumenting/writing_exporters.md

```diff
@@ -153,10 +153,10 @@ advice](/docs/practices/instrumentation/#things-to-watch-out-for) on labels.
 
 Avoid `type` as a label name, it’s too generic and meaningless. You should also
 try where possible to avoid names that are likely to clash with target labels,
-such as `region`, `zone`, `cluster`, `availability_zone`, `az`, `datacenter`,
-`dc`, `owner`, `customer`, `stage`, `environment` and `env` - though if that’s
-what the application calls something it’s best not to cause confusion by
-renaming it.
+such as `region`, `zone`, `service``cluster`, `availability_zone`, `az`,
```

Comment:
Missing `, ` after service.

Reply:
Wait, no, this is a duplicate of `service` below.

Reply:
Bah, messed up trying to group things together. Fixed.